### ISSUE TITLE
Issues with generate_legislative_index for South Korea

### ISSUE THREAD

**Issue by alexsdutton** [everypolitician/commons-builder#23]
Running `generate_legislative_index` against everypolitician/proto-commons-south-korea results in a file that only lists The National Assembly (Q494162), and none of the FLACSen or cities.

If we loosen the query to also include P1001 (applies to jurisdiction) we'd pick up a few Provincial/City Government → Province/City relationships, but…

These Provincial/City governments are modelled as instances of [local government](https://www.wikidata.org/wiki/Q6501447), not legislature or legislative house as we had before. _local government_ is also applied to e.g. [Incheon Metropolitan City Office of Education](https://www.wikidata.org/wiki/Q12613478).

I think we should add legislature types to the Provincial/City Governments, alongside their existing _local government_ types.

We can probably cope with being looseness of `?body (wdt:P194/wdt:P527?)|^wdt:P1001 ?legislature`, unless anyone thinks `wdt:P194` should be the One True Way.

http://tinyurl.com/ybey28qp provides context.

**Comment by alexsdutton** [comment]
Tagging @tmtmtmtm and @ostephens

**Comment by ostephens** [comment]
I think Government is a larger item, of which the Legislature is a part. That would point to a slightly different approach of creating (or identifying) additional items which specifically represent the legislature as part of the Government.

So we are looking for https://www.wikidata.org/wiki/Q16096697 rather than https://www.wikidata.org/wiki/Q16097618 I think

**Comment by tmtmtmtm** [comment]
Yes, in general we want legislatures, not governments. However Q16096697 was an instance of Q4358176 (council) [which isn't a subclass of legislature], so I've fix that to Q3154693 (city council). We'll probably also need to correct others too, judging by the query linked to by @ostephens above.

**Comment by ostephens** [comment]
I've changed all the metropolitan councils to be instances of Q3154693, but at the moment not obvious what the provincial councils should use.

**Comment by ostephens** [comment]
Just realised that once we add a P31 to Q37002670 (unicameral legislatures)* then this will mean these are all ultimately subclassed from legislature anyway which I think should be sufficient for g_l_i as long as other appropriate statements are in place

* or if they are bicameral (don't think they are) we'll need to do some more work to add the appropriate houses
